pkg/config/validation/validation.go

```diff
@@ -1983,6 +1983,16 @@ func validateHTTPRoute(http *networking.HTTPRoute) (errs error) {
 		}
 	}
 
+	if http.MirrorPercent != nil {
+		if http.Mirror == nil {
+			log.Warnf("mirror_percent was specified but there is no mirror field. It won't have any effect.")
+		} else if value := http.MirrorPercent.GetValue(); value > 100 {
+			errs = appendErrors(errs, fmt.Errorf("mirror_percent must have a max value of 100 (it has %d)", value))
+		}
+	}
+	if http.Mirror != nil && http.MirrorPercent == nil {
+		log.Debuga("mirror_percent not present, 100% of the traffic will be mirrored.")
```

Comment:
also skip. noisy. its already specified in api docs that it defaults to 100.

Reply:
done